bindings/c/test/mako/mako.hpp

```diff
@@ -118,6 +120,7 @@ constexpr const int NUM_CLUSTERS_MAX = 3;
 constexpr const int NUM_DATABASES_MAX = 10;
 constexpr const std::string_view KEY_PREFIX{ "mako" };
 constexpr const std::string_view TEMP_DATA_STORE{ "/tmp/makoTemp" };
+constexpr const int MAX_NUM_PATHS = 200;
```

Comment:
Can we name it something slightly more intuitive like `MAX_REPORT_FILES`?

Reply:
Sure thing